[2020-10-26 23:13:08,692] INFO Registered kafka:type=kafka.Log4jController MBean (kafka.utils.Log4jControllerRegistration$)
[2020-10-26 23:13:09,231] INFO Setting -D jdk.tls.rejectClientInitiatedRenegotiation=true to disable client-initiated TLS renegotiation (org.apache.zookeeper.common.X509Util)
[2020-10-26 23:13:09,279] INFO Registered signal handlers for TERM, INT, HUP (org.apache.kafka.common.utils.LoggingSignalHandler)
[2020-10-26 23:13:09,283] INFO starting (kafka.server.KafkaServer)
[2020-10-26 23:13:09,284] INFO Connecting to zookeeper on zookeeper:2181 (kafka.server.KafkaServer)
[2020-10-26 23:13:09,308] INFO [ZooKeeperClient Kafka server] Initializing a new session to zookeeper:2181. (kafka.zookeeper.ZooKeeperClient)
[2020-10-26 23:13:09,326] INFO Client environment:zookeeper.version=3.5.7-f0fdd52973d373ffd9c86b81d99842dc2c7f660e, built on 02/10/2020 11:30 GMT (org.apache.zookeeper.ZooKeeper)
[2020-10-26 23:13:09,329] INFO Client environment:host.name=kafka-0.kafka.kafka.svc.cluster.local (org.apache.zookeeper.ZooKeeper)
[2020-10-26 23:13:09,329] INFO Client environment:java.version=11.0.7 (org.apache.zookeeper.ZooKeeper)
[2020-10-26 23:13:09,329] INFO Client environment:java.vendor=AdoptOpenJDK (org.apache.zookeeper.ZooKeeper)
[2020-10-26 23:13:09,330] INFO Client environment:java.home=/opt/java/openjdk (org.apache.zookeeper.ZooKeeper)
[2020-10-26 23:13:09,330] INFO Client environment:java.class.path=/opt/kafka/libs/extensions/*:/opt/kafka/libs/activation-1.1.1.jar:/opt/kafka/libs/aopalliance-repackaged-2.5.0.jar:/opt/kafka/libs/argparse4j-0.7.0.jar:/opt/kafka/libs/audience-annotations-0.5.0.jar:/opt/kafka/libs/commons-cli-1.4.jar:/opt/kafka/libs/commons-lang3-3.8.1.jar:/opt/kafka/libs/connect-api-2.5.0.jar:/opt/kafka/libs/connect-basic-auth-extension-2.5.0.jar:/opt/kafka/libs/connect-file-2.5.0.jar:/opt/kafka/libs/connect-json-2.5.0.jar:/opt/kafka/libs/connect-mirror-2.5.0.jar:/opt/kafka/libs/connect-mirror-client-2.5.0.jar:/opt/kafka/libs/connect-runtime-2.5.0.jar:/opt/kafka/libs/connect-transforms-2.5.0.jar:/opt/kafka/libs/hk2-api-2.5.0.jar:/opt/kafka/libs/hk2-locator-2.5.0.jar:/opt/kafka/libs/hk2-utils-2.5.0.jar:/opt/kafka/libs/jackson-annotations-2.10.2.jar:/opt/kafka/libs/jackson-core-2.10.2.jar:/opt/kafka/libs/jackson-databind-2.10.2.jar:/opt/kafka/libs/jackson-dataformat-csv-2.10.2.jar:/opt/kafka/libs/jackson-datatype-jdk8-2.10.2.jar:/opt/kafka/libs/jackson-jaxrs-base-2.10.2.jar:/opt/kafka/libs/jackson-jaxrs-json-provider-2.10.2.jar:/opt/kafka/libs/jackson-module-jaxb-annotations-2.10.2.jar:/opt/kafka/libs/jackson-module-paranamer-2.10.2.jar:/opt/kafka/libs/jackson-module-scala_2.12-2.10.2.jar:/opt/kafka/libs/jakarta.activation-api-1.2.1.jar:/opt/kafka/libs/jakarta.annotation-api-1.3.4.jar:/opt/kafka/libs/jakarta.inject-2.5.0.jar:/opt/kafka/libs/jakarta.ws.rs-api-2.1.5.jar:/opt/kafka/libs/jakarta.xml.bind-api-2.3.2.jar:/opt/kafka/libs/javassist-3.22.0-CR2.jar:/opt/kafka/libs/javassist-3.26.0-GA.jar:/opt/kafka/libs/javax.servlet-api-3.1.0.jar:/opt/kafka/libs/javax.ws.rs-api-2.1.1.jar:/opt/kafka/libs/jaxb-api-2.3.0.jar:/opt/kafka/libs/jersey-client-2.28.jar:/opt/kafka/libs/jersey-common-2.28.jar:/opt/kafka/libs/jersey-container-servlet-2.28.jar:/opt/kafka/libs/jersey-container-servlet-core-2.28.jar:/opt/kafka/libs/jersey-hk2-2.28.jar:/opt/kafka/libs/jersey-media-jaxb-2.28.jar:/opt/kafka/libs/jersey-server-2.28.jar:/opt/kafka/libs/jetty-client-9.4.24.v20191120.jar:/opt/kafka/libs/jetty-continuation-9.4.24.v20191120.jar:/opt/kafka/libs/jetty-http-9.4.24.v20191120.jar:/opt/kafka/libs/jetty-io-9.4.24.v20191120.jar:/opt/kafka/libs/jetty-security-9.4.24.v20191120.jar:/opt/kafka/libs/jetty-server-9.4.24.v20191120.jar:/opt/kafka/libs/jetty-servlet-9.4.24.v20191120.jar:/opt/kafka/libs/jetty-servlets-9.4.24.v20191120.jar:/opt/kafka/libs/jetty-util-9.4.24.v20191120.jar:/opt/kafka/libs/jopt-simple-5.0.4.jar:/opt/kafka/libs/kafka-clients-2.5.0.jar:/opt/kafka/libs/kafka-log4j-appender-2.5.0.jar:/opt/kafka/libs/kafka-streams-2.5.0.jar:/opt/kafka/libs/kafka-streams-examples-2.5.0.jar:/opt/kafka/libs/kafka-streams-scala_2.12-2.5.0.jar:/opt/kafka/libs/kafka-streams-test-utils-2.5.0.jar:/opt/kafka/libs/kafka-tools-2.5.0.jar:/opt/kafka/libs/kafka_2.12-2.5.0-javadoc.jar:/opt/kafka/libs/kafka_2.12-2.5.0-scaladoc.jar:/opt/kafka/libs/kafka_2.12-2.5.0-sources.jar:/opt/kafka/libs/kafka_2.12-2.5.0-test-sources.jar:/opt/kafka/libs/kafka_2.12-2.5.0-test.jar:/opt/kafka/libs/kafka_2.12-2.5.0.jar:/opt/kafka/libs/log4j-1.2.17.jar:/opt/kafka/libs/lz4-java-1.7.1.jar:/opt/kafka/libs/maven-artifact-3.6.3.jar:/opt/kafka/libs/metrics-core-2.2.0.jar:/opt/kafka/libs/netty-buffer-4.1.45.Final.jar:/opt/kafka/libs/netty-codec-4.1.45.Final.jar:/opt/kafka/libs/netty-common-4.1.45.Final.jar:/opt/kafka/libs/netty-handler-4.1.45.Final.jar:/opt/kafka/libs/netty-resolver-4.1.45.Final.jar:/opt/kafka/libs/netty-transport-4.1.45.Final.jar:/opt/kafka/libs/netty-transport-native-epoll-4.1.45.Final.jar:/opt/kafka/libs/netty-transport-native-unix-common-4.1.45.Final.jar:/opt/kafka/libs/osgi-resource-locator-1.0.1.jar:/opt/kafka/libs/paranamer-2.8.jar:/opt/kafka/libs/plexus-utils-3.2.1.jar:/opt/kafka/libs/reflections-0.9.12.jar:/opt/kafka/libs/rocksdbjni-5.18.3.jar:/opt/kafka/libs/scala-java8-compat_2.12-0.9.0.jar:/opt/kafka/libs/scala-collection-compat_2.12-2.1.3.jar:/opt/kafka/libs/scala-library-2.12.10.jar:/opt/kafka/libs/scala-logging_2.12-3.9.2.jar:/opt/kafka/libs/scala-reflect-2.12.10.jar:/opt/kafka/libs/slf4j-api-1.7.30.jar:/opt/kafka/libs/slf4j-log4j12-1.7.30.jar:/opt/kafka/libs/snappy-java-1.1.7.3.jar:/opt/kafka/libs/validation-api-2.0.1.Final.jar:/opt/kafka/libs/zookeeper-3.5.7.jar:/opt/kafka/libs/zookeeper-jute-3.5.7.jar:/opt/kafka/libs/zstd-jni-1.4.4-7.jar (org.apache.zookeeper.ZooKeeper)
[2020-10-26 23:13:09,330] INFO Client environment:java.library.path=/usr/java/packages/lib:/usr/lib64:/lib64:/lib:/usr/lib (org.apache.zookeeper.ZooKeeper)
[2020-10-26 23:13:09,330] INFO Client environment:java.io.tmpdir=/tmp (org.apache.zookeeper.ZooKeeper)
[2020-10-26 23:13:09,330] INFO Client environment:java.compiler=<NA> (org.apache.zookeeper.ZooKeeper)
[2020-10-26 23:13:09,330] INFO Client environment:os.name=Linux (org.apache.zookeeper.ZooKeeper)
[2020-10-26 23:13:09,331] INFO Client environment:os.arch=amd64 (org.apache.zookeeper.ZooKeeper)
[2020-10-26 23:13:09,331] INFO Client environment:os.version=4.14.193-149.317.amzn2.x86_64 (org.apache.zookeeper.ZooKeeper)
[2020-10-26 23:13:09,331] INFO Client environment:user.name=nonroot (org.apache.zookeeper.ZooKeeper)
[2020-10-26 23:13:09,331] INFO Client environment:user.home=/home/nonroot (org.apache.zookeeper.ZooKeeper)
[2020-10-26 23:13:09,331] INFO Client environment:user.dir=/opt/kafka (org.apache.zookeeper.ZooKeeper)
[2020-10-26 23:13:09,331] INFO Client environment:os.memory.free=983MB (org.apache.zookeeper.ZooKeeper)
[2020-10-26 23:13:09,331] INFO Client environment:os.memory.max=1024MB (org.apache.zookeeper.ZooKeeper)
[2020-10-26 23:13:09,331] INFO Client environment:os.memory.total=1024MB (org.apache.zookeeper.ZooKeeper)
[2020-10-26 23:13:09,334] INFO Initiating client connection, connectString=zookeeper:2181 sessionTimeout=5000 watcher=kafka.zookeeper.ZooKeeperClient$ZooKeeperClientWatcher$@2fd953a6 (org.apache.zookeeper.ZooKeeper)
[2020-10-26 23:13:09,340] INFO jute.maxbuffer value is 4194304 Bytes (org.apache.zookeeper.ClientCnxnSocket)
[2020-10-26 23:13:09,346] INFO zookeeper.request.timeout value is 0. feature enabled= (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:13:09,351] INFO [ZooKeeperClient Kafka server] Waiting until connected. (kafka.zookeeper.ZooKeeperClient)
[2020-10-26 23:13:09,390] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:13:09,407] INFO Socket connection established, initiating session, client: /10.233.122.2:51698, server: zookeeper/10.233.2.139:2181 (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:13:09,446] INFO Session establishment complete on server zookeeper/10.233.2.139:2181, sessionid = 0x200001e82e70000, negotiated timeout = 3000 (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:13:09,449] INFO [ZooKeeperClient Kafka server] Connected. (kafka.zookeeper.ZooKeeperClient)
[2020-10-26 23:13:10,276] INFO Cluster ID = NnUC7vbTTYmufqdcPdWqMg (kafka.server.KafkaServer)
[2020-10-26 23:13:10,281] WARN No meta.properties file under dir /var/lib/kafka/data/topics/meta.properties (kafka.server.BrokerMetadataCheckpoint)
[2020-10-26 23:13:10,395] INFO KafkaConfig values: 
	advertised.host.name = null
	advertised.listeners = PLAINTEXT://:9092,OUTSIDE://10.0.2.54:32400
	advertised.port = null
	alter.config.policy.class.name = null
	alter.log.dirs.replication.quota.window.num = 11
	alter.log.dirs.replication.quota.window.size.seconds = 1
	authorizer.class.name = 
	auto.create.topics.enable = false
	auto.leader.rebalance.enable = true
	background.threads = 10
	broker.id = 0
	broker.id.generation.enable = true
	broker.rack = null
	client.quota.callback.class = null
	compression.type = producer
	connection.failed.authentication.delay.ms = 100
	connections.max.idle.ms = 600000
	connections.max.reauth.ms = 0
	control.plane.listener.name = null
	controlled.shutdown.enable = true
	controlled.shutdown.max.retries = 3
	controlled.shutdown.retry.backoff.ms = 5000
	controller.socket.timeout.ms = 5000
	create.topic.policy.class.name = null
	default.replication.factor = 3
	delegation.token.expiry.check.interval.ms = 3600000
	delegation.token.expiry.time.ms = 86400000
	delegation.token.master.key = null
	delegation.token.max.lifetime.ms = 604800000
	delete.records.purgatory.purge.interval.requests = 1
	delete.topic.enable = true
	fetch.max.bytes = 57671680
	fetch.purgatory.purge.interval.requests = 1000
	group.initial.rebalance.delay.ms = 0
	group.max.session.timeout.ms = 2000
	group.max.size = 2147483647
	group.min.session.timeout.ms = 0
	host.name = 
	inter.broker.listener.name = PLAINTEXT
	inter.broker.protocol.version = 2.5-IV0
	kafka.metrics.polling.interval.secs = 10
	kafka.metrics.reporters = []
	leader.imbalance.check.interval.seconds = 300
	leader.imbalance.per.broker.percentage = 10
	listener.security.protocol.map = PLAINTEXT:PLAINTEXT,SSL:SSL,SASL_PLAINTEXT:SASL_PLAINTEXT,SASL_SSL:SASL_SSL,OUTSIDE:PLAINTEXT
	listeners = PLAINTEXT://:9092,OUTSIDE://:9094
	log.cleaner.backoff.ms = 15000
	log.cleaner.dedupe.buffer.size = 134217728
	log.cleaner.delete.retention.ms = 86400000
	log.cleaner.enable = true
	log.cleaner.io.buffer.load.factor = 0.9
	log.cleaner.io.buffer.size = 524288
	log.cleaner.io.max.bytes.per.second = 1.7976931348623157E308
	log.cleaner.max.compaction.lag.ms = 9223372036854775807
	log.cleaner.min.cleanable.ratio = 0.5
	log.cleaner.min.compaction.lag.ms = 0
	log.cleaner.threads = 1
	log.cleanup.policy = [delete]
	log.dir = /tmp/kafka-logs
	log.dirs = /var/lib/kafka/data/topics
	log.flush.interval.messages = 9223372036854775807
	log.flush.interval.ms = null
	log.flush.offset.checkpoint.interval.ms = 60000
	log.flush.scheduler.interval.ms = 9223372036854775807
	log.flush.start.offset.checkpoint.interval.ms = 60000
	log.index.interval.bytes = 4096
	log.index.size.max.bytes = 10485760
	log.message.downconversion.enable = true
	log.message.format.version = 2.5-IV0
	log.message.timestamp.difference.max.ms = 9223372036854775807
	log.message.timestamp.type = CreateTime
	log.preallocate = false
	log.retention.bytes = -1
	log.retention.check.interval.ms = 300000
	log.retention.hours = -1
	log.retention.minutes = null
	log.retention.ms = null
	log.roll.hours = 168
	log.roll.jitter.hours = 0
	log.roll.jitter.ms = null
	log.roll.ms = null
	log.segment.bytes = 1073741824
	log.segment.delete.delay.ms = 60000
	max.connections = 2147483647
	max.connections.per.ip = 2147483647
	max.connections.per.ip.overrides = 
	max.incremental.fetch.session.cache.slots = 1000
	message.max.bytes = 1048588
	metric.reporters = []
	metrics.num.samples = 2
	metrics.recording.level = INFO
	metrics.sample.window.ms = 30000
	min.insync.replicas = 2
	num.io.threads = 8
	num.network.threads = 3
	num.partitions = 1
	num.recovery.threads.per.data.dir = 1
	num.replica.alter.log.dirs.threads = null
	num.replica.fetchers = 2
	offset.metadata.max.bytes = 4096
	offsets.commit.required.acks = -1
	offsets.commit.timeout.ms = 5000
	offsets.load.buffer.size = 5242880
	offsets.retention.check.interval.ms = 600000
	offsets.retention.minutes = 10080
	offsets.topic.compression.codec = 0
	offsets.topic.num.partitions = 2
	offsets.topic.replication.factor = 5
	offsets.topic.segment.bytes = 104857600
	password.encoder.cipher.algorithm = AES/CBC/PKCS5Padding
	password.encoder.iterations = 4096
	password.encoder.key.length = 128
	password.encoder.keyfactory.algorithm = null
	password.encoder.old.secret = null
	password.encoder.secret = null
	port = 9092
	principal.builder.class = null
	producer.purgatory.purge.interval.requests = 1000
	queued.max.request.bytes = -1
	queued.max.requests = 500
	quota.consumer.default = 9223372036854775807
	quota.producer.default = 9223372036854775807
	quota.window.num = 11
	quota.window.size.seconds = 1
	replica.fetch.backoff.ms = 1000
	replica.fetch.max.bytes = 1048576
	replica.fetch.min.bytes = 1
	replica.fetch.response.max.bytes = 10485760
	replica.fetch.wait.max.ms = 500
	replica.high.watermark.checkpoint.interval.ms = 5000
	replica.lag.time.max.ms = 5000
	replica.selector.class = null
	replica.socket.receive.buffer.bytes = 65536
	replica.socket.timeout.ms = 3000
	replication.quota.window.num = 11
	replication.quota.window.size.seconds = 1
	request.timeout.ms = 2000
	reserved.broker.max.id = 1000
	sasl.client.callback.handler.class = null
	sasl.enabled.mechanisms = [GSSAPI]
	sasl.jaas.config = null
	sasl.kerberos.kinit.cmd = /usr/bin/kinit
	sasl.kerberos.min.time.before.relogin = 60000
	sasl.kerberos.principal.to.local.rules = [DEFAULT]
	sasl.kerberos.service.name = null
	sasl.kerberos.ticket.renew.jitter = 0.05
	sasl.kerberos.ticket.renew.window.factor = 0.8
	sasl.login.callback.handler.class = null
	sasl.login.class = null
	sasl.login.refresh.buffer.seconds = 300
	sasl.login.refresh.min.period.seconds = 60
	sasl.login.refresh.window.factor = 0.8
	sasl.login.refresh.window.jitter = 0.05
	sasl.mechanism.inter.broker.protocol = GSSAPI
	sasl.server.callback.handler.class = null
	security.inter.broker.protocol = PLAINTEXT
	security.providers = null
	socket.receive.buffer.bytes = 102400
	socket.request.max.bytes = 104857600
	socket.send.buffer.bytes = 102400
	ssl.cipher.suites = []
	ssl.client.auth = none
	ssl.enabled.protocols = [TLSv1.2]
	ssl.endpoint.identification.algorithm = https
	ssl.key.password = null
	ssl.keymanager.algorithm = SunX509
	ssl.keystore.location = null
	ssl.keystore.password = null
	ssl.keystore.type = JKS
	ssl.principal.mapping.rules = DEFAULT
	ssl.protocol = TLSv1.2
	ssl.provider = null
	ssl.secure.random.implementation = null
	ssl.trustmanager.algorithm = PKIX
	ssl.truststore.location = null
	ssl.truststore.password = null
	ssl.truststore.type = JKS
	transaction.abort.timed.out.transaction.cleanup.interval.ms = 10000
	transaction.max.timeout.ms = 3000
	transaction.remove.expired.transaction.cleanup.interval.ms = 3600000
	transaction.state.log.load.buffer.size = 5242880
	transaction.state.log.min.isr = 2
	transaction.state.log.num.partitions = 50
	transaction.state.log.replication.factor = 3
	transaction.state.log.segment.bytes = 104857600
	transactional.id.expiration.ms = 604800000
	unclean.leader.election.enable = false
	zookeeper.clientCnxnSocket = null
	zookeeper.connect = zookeeper:2181
	zookeeper.connection.timeout.ms = 3000
	zookeeper.max.in.flight.requests = 10
	zookeeper.session.timeout.ms = 5000
	zookeeper.set.acl = false
	zookeeper.ssl.cipher.suites = null
	zookeeper.ssl.client.enable = false
	zookeeper.ssl.crl.enable = false
	zookeeper.ssl.enabled.protocols = null
	zookeeper.ssl.endpoint.identification.algorithm = HTTPS
	zookeeper.ssl.keystore.location = null
	zookeeper.ssl.keystore.password = null
	zookeeper.ssl.keystore.type = null
	zookeeper.ssl.ocsp.enable = false
	zookeeper.ssl.protocol = TLSv1.2
	zookeeper.ssl.truststore.location = null
	zookeeper.ssl.truststore.password = null
	zookeeper.ssl.truststore.type = null
	zookeeper.sync.time.ms = 2000
 (kafka.server.KafkaConfig)
[2020-10-26 23:13:10,416] INFO KafkaConfig values: 
	advertised.host.name = null
	advertised.listeners = PLAINTEXT://:9092,OUTSIDE://10.0.2.54:32400
	advertised.port = null
	alter.config.policy.class.name = null
	alter.log.dirs.replication.quota.window.num = 11
	alter.log.dirs.replication.quota.window.size.seconds = 1
	authorizer.class.name = 
	auto.create.topics.enable = false
	auto.leader.rebalance.enable = true
	background.threads = 10
	broker.id = 0
	broker.id.generation.enable = true
	broker.rack = null
	client.quota.callback.class = null
	compression.type = producer
	connection.failed.authentication.delay.ms = 100
	connections.max.idle.ms = 600000
	connections.max.reauth.ms = 0
	control.plane.listener.name = null
	controlled.shutdown.enable = true
	controlled.shutdown.max.retries = 3
	controlled.shutdown.retry.backoff.ms = 5000
	controller.socket.timeout.ms = 5000
	create.topic.policy.class.name = null
	default.replication.factor = 3
	delegation.token.expiry.check.interval.ms = 3600000
	delegation.token.expiry.time.ms = 86400000
	delegation.token.master.key = null
	delegation.token.max.lifetime.ms = 604800000
	delete.records.purgatory.purge.interval.requests = 1
	delete.topic.enable = true
	fetch.max.bytes = 57671680
	fetch.purgatory.purge.interval.requests = 1000
	group.initial.rebalance.delay.ms = 0
	group.max.session.timeout.ms = 2000
	group.max.size = 2147483647
	group.min.session.timeout.ms = 0
	host.name = 
	inter.broker.listener.name = PLAINTEXT
	inter.broker.protocol.version = 2.5-IV0
	kafka.metrics.polling.interval.secs = 10
	kafka.metrics.reporters = []
	leader.imbalance.check.interval.seconds = 300
	leader.imbalance.per.broker.percentage = 10
	listener.security.protocol.map = PLAINTEXT:PLAINTEXT,SSL:SSL,SASL_PLAINTEXT:SASL_PLAINTEXT,SASL_SSL:SASL_SSL,OUTSIDE:PLAINTEXT
	listeners = PLAINTEXT://:9092,OUTSIDE://:9094
	log.cleaner.backoff.ms = 15000
	log.cleaner.dedupe.buffer.size = 134217728
	log.cleaner.delete.retention.ms = 86400000
	log.cleaner.enable = true
	log.cleaner.io.buffer.load.factor = 0.9
	log.cleaner.io.buffer.size = 524288
	log.cleaner.io.max.bytes.per.second = 1.7976931348623157E308
	log.cleaner.max.compaction.lag.ms = 9223372036854775807
	log.cleaner.min.cleanable.ratio = 0.5
	log.cleaner.min.compaction.lag.ms = 0
	log.cleaner.threads = 1
	log.cleanup.policy = [delete]
	log.dir = /tmp/kafka-logs
	log.dirs = /var/lib/kafka/data/topics
	log.flush.interval.messages = 9223372036854775807
	log.flush.interval.ms = null
	log.flush.offset.checkpoint.interval.ms = 60000
	log.flush.scheduler.interval.ms = 9223372036854775807
	log.flush.start.offset.checkpoint.interval.ms = 60000
	log.index.interval.bytes = 4096
	log.index.size.max.bytes = 10485760
	log.message.downconversion.enable = true
	log.message.format.version = 2.5-IV0
	log.message.timestamp.difference.max.ms = 9223372036854775807
	log.message.timestamp.type = CreateTime
	log.preallocate = false
	log.retention.bytes = -1
	log.retention.check.interval.ms = 300000
	log.retention.hours = -1
	log.retention.minutes = null
	log.retention.ms = null
	log.roll.hours = 168
	log.roll.jitter.hours = 0
	log.roll.jitter.ms = null
	log.roll.ms = null
	log.segment.bytes = 1073741824
	log.segment.delete.delay.ms = 60000
	max.connections = 2147483647
	max.connections.per.ip = 2147483647
	max.connections.per.ip.overrides = 
	max.incremental.fetch.session.cache.slots = 1000
	message.max.bytes = 1048588
	metric.reporters = []
	metrics.num.samples = 2
	metrics.recording.level = INFO
	metrics.sample.window.ms = 30000
	min.insync.replicas = 2
	num.io.threads = 8
	num.network.threads = 3
	num.partitions = 1
	num.recovery.threads.per.data.dir = 1
	num.replica.alter.log.dirs.threads = null
	num.replica.fetchers = 2
	offset.metadata.max.bytes = 4096
	offsets.commit.required.acks = -1
	offsets.commit.timeout.ms = 5000
	offsets.load.buffer.size = 5242880
	offsets.retention.check.interval.ms = 600000
	offsets.retention.minutes = 10080
	offsets.topic.compression.codec = 0
	offsets.topic.num.partitions = 2
	offsets.topic.replication.factor = 5
	offsets.topic.segment.bytes = 104857600
	password.encoder.cipher.algorithm = AES/CBC/PKCS5Padding
	password.encoder.iterations = 4096
	password.encoder.key.length = 128
	password.encoder.keyfactory.algorithm = null
	password.encoder.old.secret = null
	password.encoder.secret = null
	port = 9092
	principal.builder.class = null
	producer.purgatory.purge.interval.requests = 1000
	queued.max.request.bytes = -1
	queued.max.requests = 500
	quota.consumer.default = 9223372036854775807
	quota.producer.default = 9223372036854775807
	quota.window.num = 11
	quota.window.size.seconds = 1
	replica.fetch.backoff.ms = 1000
	replica.fetch.max.bytes = 1048576
	replica.fetch.min.bytes = 1
	replica.fetch.response.max.bytes = 10485760
	replica.fetch.wait.max.ms = 500
	replica.high.watermark.checkpoint.interval.ms = 5000
	replica.lag.time.max.ms = 5000
	replica.selector.class = null
	replica.socket.receive.buffer.bytes = 65536
	replica.socket.timeout.ms = 3000
	replication.quota.window.num = 11
	replication.quota.window.size.seconds = 1
	request.timeout.ms = 2000
	reserved.broker.max.id = 1000
	sasl.client.callback.handler.class = null
	sasl.enabled.mechanisms = [GSSAPI]
	sasl.jaas.config = null
	sasl.kerberos.kinit.cmd = /usr/bin/kinit
	sasl.kerberos.min.time.before.relogin = 60000
	sasl.kerberos.principal.to.local.rules = [DEFAULT]
	sasl.kerberos.service.name = null
	sasl.kerberos.ticket.renew.jitter = 0.05
	sasl.kerberos.ticket.renew.window.factor = 0.8
	sasl.login.callback.handler.class = null
	sasl.login.class = null
	sasl.login.refresh.buffer.seconds = 300
	sasl.login.refresh.min.period.seconds = 60
	sasl.login.refresh.window.factor = 0.8
	sasl.login.refresh.window.jitter = 0.05
	sasl.mechanism.inter.broker.protocol = GSSAPI
	sasl.server.callback.handler.class = null
	security.inter.broker.protocol = PLAINTEXT
	security.providers = null
	socket.receive.buffer.bytes = 102400
	socket.request.max.bytes = 104857600
	socket.send.buffer.bytes = 102400
	ssl.cipher.suites = []
	ssl.client.auth = none
	ssl.enabled.protocols = [TLSv1.2]
	ssl.endpoint.identification.algorithm = https
	ssl.key.password = null
	ssl.keymanager.algorithm = SunX509
	ssl.keystore.location = null
	ssl.keystore.password = null
	ssl.keystore.type = JKS
	ssl.principal.mapping.rules = DEFAULT
	ssl.protocol = TLSv1.2
	ssl.provider = null
	ssl.secure.random.implementation = null
	ssl.trustmanager.algorithm = PKIX
	ssl.truststore.location = null
	ssl.truststore.password = null
	ssl.truststore.type = JKS
	transaction.abort.timed.out.transaction.cleanup.interval.ms = 10000
	transaction.max.timeout.ms = 3000
	transaction.remove.expired.transaction.cleanup.interval.ms = 3600000
	transaction.state.log.load.buffer.size = 5242880
	transaction.state.log.min.isr = 2
	transaction.state.log.num.partitions = 50
	transaction.state.log.replication.factor = 3
	transaction.state.log.segment.bytes = 104857600
	transactional.id.expiration.ms = 604800000
	unclean.leader.election.enable = false
	zookeeper.clientCnxnSocket = null
	zookeeper.connect = zookeeper:2181
	zookeeper.connection.timeout.ms = 3000
	zookeeper.max.in.flight.requests = 10
	zookeeper.session.timeout.ms = 5000
	zookeeper.set.acl = false
	zookeeper.ssl.cipher.suites = null
	zookeeper.ssl.client.enable = false
	zookeeper.ssl.crl.enable = false
	zookeeper.ssl.enabled.protocols = null
	zookeeper.ssl.endpoint.identification.algorithm = HTTPS
	zookeeper.ssl.keystore.location = null
	zookeeper.ssl.keystore.password = null
	zookeeper.ssl.keystore.type = null
	zookeeper.ssl.ocsp.enable = false
	zookeeper.ssl.protocol = TLSv1.2
	zookeeper.ssl.truststore.location = null
	zookeeper.ssl.truststore.password = null
	zookeeper.ssl.truststore.type = null
	zookeeper.sync.time.ms = 2000
 (kafka.server.KafkaConfig)
[2020-10-26 23:13:10,451] INFO [ThrottledChannelReaper-Fetch]: Starting (kafka.server.ClientQuotaManager$ThrottledChannelReaper)
[2020-10-26 23:13:10,459] INFO [ThrottledChannelReaper-Request]: Starting (kafka.server.ClientQuotaManager$ThrottledChannelReaper)
[2020-10-26 23:13:10,451] INFO [ThrottledChannelReaper-Produce]: Starting (kafka.server.ClientQuotaManager$ThrottledChannelReaper)
[2020-10-26 23:13:10,488] INFO Log directory /var/lib/kafka/data/topics not found, creating it. (kafka.log.LogManager)
[2020-10-26 23:13:10,495] INFO Loading logs. (kafka.log.LogManager)
[2020-10-26 23:13:10,508] INFO Logs loading complete in 13 ms. (kafka.log.LogManager)
[2020-10-26 23:13:10,525] INFO Starting log cleanup with a period of 300000 ms. (kafka.log.LogManager)
[2020-10-26 23:13:10,528] INFO Starting log flusher with a default period of 9223372036854775807 ms. (kafka.log.LogManager)
[2020-10-26 23:13:11,149] INFO Awaiting socket connections on 0.0.0.0:9092. (kafka.network.Acceptor)
[2020-10-26 23:13:11,200] INFO [SocketServer brokerId=0] Created data-plane acceptor and processors for endpoint : EndPoint(null,9092,ListenerName(PLAINTEXT),PLAINTEXT) (kafka.network.SocketServer)
[2020-10-26 23:13:11,219] INFO Awaiting socket connections on 0.0.0.0:9094. (kafka.network.Acceptor)
[2020-10-26 23:13:11,270] INFO [SocketServer brokerId=0] Created data-plane acceptor and processors for endpoint : EndPoint(null,9094,ListenerName(OUTSIDE),PLAINTEXT) (kafka.network.SocketServer)
[2020-10-26 23:13:11,272] INFO [SocketServer brokerId=0] Started 2 acceptor threads for data-plane (kafka.network.SocketServer)
[2020-10-26 23:13:11,318] INFO [ExpirationReaper-0-Produce]: Starting (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2020-10-26 23:13:11,322] INFO [ExpirationReaper-0-DeleteRecords]: Starting (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2020-10-26 23:13:11,319] INFO [ExpirationReaper-0-Fetch]: Starting (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2020-10-26 23:13:11,325] INFO [ExpirationReaper-0-ElectLeader]: Starting (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2020-10-26 23:13:11,348] INFO [LogDirFailureHandler]: Starting (kafka.server.ReplicaManager$LogDirFailureHandler)
[2020-10-26 23:13:11,431] INFO Creating /brokers/ids/0 (is it secure? false) (kafka.zk.KafkaZkClient)
[2020-10-26 23:13:11,486] INFO Stat of the created znode at /brokers/ids/0 is: 4294967406,4294967406,1603753991451,1603753991451,1,0,0,144115319121051648,294,0,4294967406
 (kafka.zk.KafkaZkClient)
[2020-10-26 23:13:11,487] INFO Registered broker 0 at path /brokers/ids/0 with addresses: ArrayBuffer(EndPoint(kafka-0.kafka.kafka.svc.cluster.local,9092,ListenerName(PLAINTEXT),PLAINTEXT), EndPoint(10.0.2.54,32400,ListenerName(OUTSIDE),PLAINTEXT)), czxid (broker epoch): 4294967406 (kafka.zk.KafkaZkClient)
[2020-10-26 23:13:11,582] INFO [ExpirationReaper-0-topic]: Starting (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2020-10-26 23:13:11,598] INFO [ExpirationReaper-0-Heartbeat]: Starting (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2020-10-26 23:13:11,599] INFO [ExpirationReaper-0-Rebalance]: Starting (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2020-10-26 23:13:11,632] INFO [GroupCoordinator 0]: Starting up. (kafka.coordinator.group.GroupCoordinator)
[2020-10-26 23:13:11,633] INFO [GroupCoordinator 0]: Startup complete. (kafka.coordinator.group.GroupCoordinator)
[2020-10-26 23:13:11,646] INFO [GroupMetadataManager brokerId=0] Removed 0 expired offsets in 11 milliseconds. (kafka.coordinator.group.GroupMetadataManager)
[2020-10-26 23:13:11,740] INFO [ProducerId Manager 0]: Acquired new producerId block (brokerId:0,blockStartProducerId:5000,blockEndProducerId:5999) by writing to Zk with path version 6 (kafka.coordinator.transaction.ProducerIdManager)
[2020-10-26 23:13:11,795] INFO [TransactionCoordinator id=0] Starting up. (kafka.coordinator.transaction.TransactionCoordinator)
[2020-10-26 23:13:11,797] INFO [TransactionCoordinator id=0] Startup complete. (kafka.coordinator.transaction.TransactionCoordinator)
[2020-10-26 23:13:11,816] INFO [Transaction Marker Channel Manager 0]: Starting (kafka.coordinator.transaction.TransactionMarkerChannelManager)
[2020-10-26 23:13:11,839] INFO [ExpirationReaper-0-AlterAcls]: Starting (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2020-10-26 23:13:11,900] INFO [/config/changes-event-process-thread]: Starting (kafka.common.ZkNodeChangeNotificationListener$ChangeEventProcessThread)
[2020-10-26 23:13:11,964] INFO [SocketServer brokerId=0] Started data-plane processors for 2 acceptors (kafka.network.SocketServer)
[2020-10-26 23:13:11,978] INFO Kafka version: 2.5.0 (org.apache.kafka.common.utils.AppInfoParser)
[2020-10-26 23:13:11,978] INFO Kafka commitId: 66563e712b0b9f84 (org.apache.kafka.common.utils.AppInfoParser)
[2020-10-26 23:13:11,978] INFO Kafka startTimeMs: 1603753991969 (org.apache.kafka.common.utils.AppInfoParser)
[2020-10-26 23:13:11,984] INFO [KafkaServer id=0] started (kafka.server.KafkaServer)
[2020-10-26 23:14:27,866] INFO Unable to read additional data from server sessionid 0x200001e82e70000, likely server has closed socket, closing socket connection and attempting reconnect (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:14:29,589] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:14:29,598] INFO Socket connection established, initiating session, client: /10.233.122.2:52062, server: zookeeper/10.233.2.139:2181 (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:14:29,620] INFO Session establishment complete on server zookeeper/10.233.2.139:2181, sessionid = 0x200001e82e70000, negotiated timeout = 3000 (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:15:40,649] INFO [ReplicaFetcherManager on broker 0] Removed fetcher for partitions Set(t1_filtered-0) (kafka.server.ReplicaFetcherManager)
[2020-10-26 23:15:40,939] INFO [Log partition=t1_filtered-0, dir=/var/lib/kafka/data/topics] Loading producer state till offset 0 with message format version 2 (kafka.log.Log)
[2020-10-26 23:15:40,970] INFO [Log partition=t1_filtered-0, dir=/var/lib/kafka/data/topics] Completed load of log with 1 segments, log start offset 0 and log end offset 0 in 170 ms (kafka.log.Log)
[2020-10-26 23:15:40,973] INFO Created log for partition t1_filtered-0 in /var/lib/kafka/data/topics/t1_filtered-0 with properties {compression.type -> producer, min.insync.replicas -> 2, message.downconversion.enable -> true, segment.jitter.ms -> 0, cleanup.policy -> [delete], flush.ms -> 9223372036854775807, retention.ms -> -1, segment.bytes -> 1073741824, flush.messages -> 9223372036854775807, message.format.version -> 2.5-IV0, max.compaction.lag.ms -> 9223372036854775807, file.delete.delay.ms -> 60000, max.message.bytes -> 1048588, min.compaction.lag.ms -> 0, message.timestamp.type -> CreateTime, preallocate -> false, index.interval.bytes -> 4096, min.cleanable.dirty.ratio -> 0.5, unclean.leader.election.enable -> false, retention.bytes -> -1, delete.retention.ms -> 86400000, segment.ms -> 604800000, message.timestamp.difference.max.ms -> 9223372036854775807, segment.index.bytes -> 10485760}. (kafka.log.LogManager)
[2020-10-26 23:15:40,997] INFO [Partition t1_filtered-0 broker=0] No checkpointed highwatermark is found for partition t1_filtered-0 (kafka.cluster.Partition)
[2020-10-26 23:15:40,998] INFO [Partition t1_filtered-0 broker=0] Log loaded for partition t1_filtered-0 with initial high watermark 0 (kafka.cluster.Partition)
[2020-10-26 23:15:41,008] INFO [Partition t1_filtered-0 broker=0] t1_filtered-0 starts at leader epoch 0 from offset 0 with high watermark 0. Previous leader epoch was -1. (kafka.cluster.Partition)
[2020-10-26 23:15:46,488] INFO [Log partition=t4_warnings-0, dir=/var/lib/kafka/data/topics] Loading producer state till offset 0 with message format version 2 (kafka.log.Log)
[2020-10-26 23:15:46,490] INFO [Log partition=t4_warnings-0, dir=/var/lib/kafka/data/topics] Completed load of log with 1 segments, log start offset 0 and log end offset 0 in 4 ms (kafka.log.Log)
[2020-10-26 23:15:46,494] INFO Created log for partition t4_warnings-0 in /var/lib/kafka/data/topics/t4_warnings-0 with properties {compression.type -> producer, min.insync.replicas -> 2, message.downconversion.enable -> true, segment.jitter.ms -> 0, cleanup.policy -> [delete], flush.ms -> 9223372036854775807, retention.ms -> -1, segment.bytes -> 1073741824, flush.messages -> 9223372036854775807, message.format.version -> 2.5-IV0, max.compaction.lag.ms -> 9223372036854775807, file.delete.delay.ms -> 60000, max.message.bytes -> 1048588, min.compaction.lag.ms -> 0, message.timestamp.type -> CreateTime, preallocate -> false, index.interval.bytes -> 4096, min.cleanable.dirty.ratio -> 0.5, unclean.leader.election.enable -> false, retention.bytes -> -1, delete.retention.ms -> 86400000, segment.ms -> 604800000, message.timestamp.difference.max.ms -> 9223372036854775807, segment.index.bytes -> 10485760}. (kafka.log.LogManager)
[2020-10-26 23:15:46,495] INFO [Partition t4_warnings-0 broker=0] No checkpointed highwatermark is found for partition t4_warnings-0 (kafka.cluster.Partition)
[2020-10-26 23:15:46,499] INFO [Partition t4_warnings-0 broker=0] Log loaded for partition t4_warnings-0 with initial high watermark 0 (kafka.cluster.Partition)
[2020-10-26 23:15:46,503] INFO [ReplicaFetcherManager on broker 0] Removed fetcher for partitions Set(t4_warnings-0) (kafka.server.ReplicaFetcherManager)
[2020-10-26 23:15:46,536] INFO [ReplicaFetcherManager on broker 0] Added fetcher to broker 1 for partitions Map(t4_warnings-0 -> (offset=0, leaderEpoch=0)) (kafka.server.ReplicaFetcherManager)
[2020-10-26 23:15:46,551] INFO [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Starting (kafka.server.ReplicaFetcherThread)
[2020-10-26 23:15:46,554] INFO [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Truncating partition t4_warnings-0 to local high watermark 0 (kafka.server.ReplicaFetcherThread)
[2020-10-26 23:15:46,555] INFO [Log partition=t4_warnings-0, dir=/var/lib/kafka/data/topics] Truncating to 0 has no effect as the largest offset in the log is -1 (kafka.log.Log)
[2020-10-26 23:15:49,287] INFO [Log partition=t5_filtered-0, dir=/var/lib/kafka/data/topics] Loading producer state till offset 0 with message format version 2 (kafka.log.Log)
[2020-10-26 23:15:49,288] INFO [Log partition=t5_filtered-0, dir=/var/lib/kafka/data/topics] Completed load of log with 1 segments, log start offset 0 and log end offset 0 in 2 ms (kafka.log.Log)
[2020-10-26 23:15:49,289] INFO Created log for partition t5_filtered-0 in /var/lib/kafka/data/topics/t5_filtered-0 with properties {compression.type -> producer, min.insync.replicas -> 2, message.downconversion.enable -> true, segment.jitter.ms -> 0, cleanup.policy -> [delete], flush.ms -> 9223372036854775807, retention.ms -> -1, segment.bytes -> 1073741824, flush.messages -> 9223372036854775807, message.format.version -> 2.5-IV0, max.compaction.lag.ms -> 9223372036854775807, file.delete.delay.ms -> 60000, max.message.bytes -> 1048588, min.compaction.lag.ms -> 0, message.timestamp.type -> CreateTime, preallocate -> false, index.interval.bytes -> 4096, min.cleanable.dirty.ratio -> 0.5, unclean.leader.election.enable -> false, retention.bytes -> -1, delete.retention.ms -> 86400000, segment.ms -> 604800000, message.timestamp.difference.max.ms -> 9223372036854775807, segment.index.bytes -> 10485760}. (kafka.log.LogManager)
[2020-10-26 23:15:49,289] INFO [Partition t5_filtered-0 broker=0] No checkpointed highwatermark is found for partition t5_filtered-0 (kafka.cluster.Partition)
[2020-10-26 23:15:49,290] INFO [Partition t5_filtered-0 broker=0] Log loaded for partition t5_filtered-0 with initial high watermark 0 (kafka.cluster.Partition)
[2020-10-26 23:15:49,291] INFO [ReplicaFetcherManager on broker 0] Removed fetcher for partitions Set(t5_filtered-0) (kafka.server.ReplicaFetcherManager)
[2020-10-26 23:15:49,295] INFO [ReplicaFetcherManager on broker 0] Added fetcher to broker 1 for partitions Map(t5_filtered-0 -> (offset=0, leaderEpoch=0)) (kafka.server.ReplicaFetcherManager)
[2020-10-26 23:15:49,297] INFO [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=1] Starting (kafka.server.ReplicaFetcherThread)
[2020-10-26 23:15:49,297] INFO [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=1] Truncating partition t5_filtered-0 to local high watermark 0 (kafka.server.ReplicaFetcherThread)
[2020-10-26 23:15:49,298] INFO [Log partition=t5_filtered-0, dir=/var/lib/kafka/data/topics] Truncating to 0 has no effect as the largest offset in the log is -1 (kafka.log.Log)
[2020-10-26 23:16:44,915] INFO [Log partition=t1-0, dir=/var/lib/kafka/data/topics] Loading producer state till offset 0 with message format version 2 (kafka.log.Log)
[2020-10-26 23:16:44,915] INFO [Log partition=t1-0, dir=/var/lib/kafka/data/topics] Completed load of log with 1 segments, log start offset 0 and log end offset 0 in 3 ms (kafka.log.Log)
[2020-10-26 23:16:44,917] INFO Created log for partition t1-0 in /var/lib/kafka/data/topics/t1-0 with properties {compression.type -> producer, min.insync.replicas -> 2, message.downconversion.enable -> true, segment.jitter.ms -> 0, cleanup.policy -> [delete], flush.ms -> 9223372036854775807, retention.ms -> -1, segment.bytes -> 1073741824, flush.messages -> 9223372036854775807, message.format.version -> 2.5-IV0, max.compaction.lag.ms -> 9223372036854775807, file.delete.delay.ms -> 60000, max.message.bytes -> 1048588, min.compaction.lag.ms -> 0, message.timestamp.type -> CreateTime, preallocate -> false, index.interval.bytes -> 4096, min.cleanable.dirty.ratio -> 0.5, unclean.leader.election.enable -> false, retention.bytes -> -1, delete.retention.ms -> 86400000, segment.ms -> 604800000, message.timestamp.difference.max.ms -> 9223372036854775807, segment.index.bytes -> 10485760}. (kafka.log.LogManager)
[2020-10-26 23:16:44,919] INFO [Partition t1-0 broker=0] No checkpointed highwatermark is found for partition t1-0 (kafka.cluster.Partition)
[2020-10-26 23:16:44,919] INFO [Partition t1-0 broker=0] Log loaded for partition t1-0 with initial high watermark 0 (kafka.cluster.Partition)
[2020-10-26 23:16:44,920] INFO [ReplicaFetcherManager on broker 0] Removed fetcher for partitions Set(t1-0) (kafka.server.ReplicaFetcherManager)
[2020-10-26 23:16:44,929] INFO [ReplicaFetcherManager on broker 0] Added fetcher to broker 4 for partitions Map(t1-0 -> (offset=0, leaderEpoch=0)) (kafka.server.ReplicaFetcherManager)
[2020-10-26 23:16:44,933] INFO [ReplicaFetcher replicaId=0, leaderId=4, fetcherId=1] Starting (kafka.server.ReplicaFetcherThread)
[2020-10-26 23:16:44,934] INFO [ReplicaFetcher replicaId=0, leaderId=4, fetcherId=1] Truncating partition t1-0 to local high watermark 0 (kafka.server.ReplicaFetcherThread)
[2020-10-26 23:16:44,934] INFO [Log partition=t1-0, dir=/var/lib/kafka/data/topics] Truncating to 0 has no effect as the largest offset in the log is -1 (kafka.log.Log)
[2020-10-26 23:16:52,341] INFO [Log partition=t4-0, dir=/var/lib/kafka/data/topics] Loading producer state till offset 0 with message format version 2 (kafka.log.Log)
[2020-10-26 23:16:52,341] INFO [Log partition=t4-0, dir=/var/lib/kafka/data/topics] Completed load of log with 1 segments, log start offset 0 and log end offset 0 in 1 ms (kafka.log.Log)
[2020-10-26 23:16:52,342] INFO Created log for partition t4-0 in /var/lib/kafka/data/topics/t4-0 with properties {compression.type -> producer, min.insync.replicas -> 2, message.downconversion.enable -> true, segment.jitter.ms -> 0, cleanup.policy -> [delete], flush.ms -> 9223372036854775807, retention.ms -> -1, segment.bytes -> 1073741824, flush.messages -> 9223372036854775807, message.format.version -> 2.5-IV0, max.compaction.lag.ms -> 9223372036854775807, file.delete.delay.ms -> 60000, max.message.bytes -> 1048588, min.compaction.lag.ms -> 0, message.timestamp.type -> CreateTime, preallocate -> false, index.interval.bytes -> 4096, min.cleanable.dirty.ratio -> 0.5, unclean.leader.election.enable -> false, retention.bytes -> -1, delete.retention.ms -> 86400000, segment.ms -> 604800000, message.timestamp.difference.max.ms -> 9223372036854775807, segment.index.bytes -> 10485760}. (kafka.log.LogManager)
[2020-10-26 23:16:52,343] INFO [Partition t4-0 broker=0] No checkpointed highwatermark is found for partition t4-0 (kafka.cluster.Partition)
[2020-10-26 23:16:52,343] INFO [Partition t4-0 broker=0] Log loaded for partition t4-0 with initial high watermark 0 (kafka.cluster.Partition)
[2020-10-26 23:16:52,343] INFO [ReplicaFetcherManager on broker 0] Removed fetcher for partitions Set(t4-0) (kafka.server.ReplicaFetcherManager)
[2020-10-26 23:16:52,351] INFO [ReplicaFetcherManager on broker 0] Added fetcher to broker 3 for partitions Map(t4-0 -> (offset=0, leaderEpoch=0)) (kafka.server.ReplicaFetcherManager)
[2020-10-26 23:16:52,353] INFO [ReplicaFetcher replicaId=0, leaderId=3, fetcherId=0] Starting (kafka.server.ReplicaFetcherThread)
[2020-10-26 23:16:52,354] INFO [ReplicaFetcher replicaId=0, leaderId=3, fetcherId=0] Truncating partition t4-0 to local high watermark 0 (kafka.server.ReplicaFetcherThread)
[2020-10-26 23:16:52,355] INFO [Log partition=t4-0, dir=/var/lib/kafka/data/topics] Truncating to 0 has no effect as the largest offset in the log is -1 (kafka.log.Log)
[2020-10-26 23:18:10,514] INFO Unable to read additional data from server sessionid 0x200001e82e70000, likely server has closed socket, closing socket connection and attempting reconnect (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:18:12,639] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:18:12,647] INFO Socket connection established, initiating session, client: /10.233.122.2:52976, server: zookeeper/10.233.2.139:2181 (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:18:12,690] WARN Unable to reconnect to ZooKeeper service, session 0x200001e82e70000 has expired (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:18:12,690] INFO Unable to reconnect to ZooKeeper service, session 0x200001e82e70000 has expired, closing socket connection (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:18:12,691] INFO EventThread shut down for session: 0x200001e82e70000 (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:18:12,692] INFO [ZooKeeperClient Kafka server] Session expired. (kafka.zookeeper.ZooKeeperClient)
[2020-10-26 23:18:12,694] INFO [ZooKeeperClient Kafka server] Initializing a new session to zookeeper:2181. (kafka.zookeeper.ZooKeeperClient)
[2020-10-26 23:18:12,694] INFO Initiating client connection, connectString=zookeeper:2181 sessionTimeout=5000 watcher=kafka.zookeeper.ZooKeeperClient$ZooKeeperClientWatcher$@2fd953a6 (org.apache.zookeeper.ZooKeeper)
[2020-10-26 23:18:12,701] INFO jute.maxbuffer value is 4194304 Bytes (org.apache.zookeeper.ClientCnxnSocket)
[2020-10-26 23:18:12,702] INFO zookeeper.request.timeout value is 0. feature enabled= (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:18:12,711] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:18:12,715] INFO Creating /brokers/ids/0 (is it secure? false) (kafka.zk.KafkaZkClient)
[2020-10-26 23:18:12,724] INFO Socket connection established, initiating session, client: /10.233.122.2:52978, server: zookeeper/10.233.2.139:2181 (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:18:12,775] INFO [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=1] Partition t5_filtered-0 has an older epoch (0) than the current leader. Will await the new LeaderAndIsr state before resuming fetching. (kafka.server.ReplicaFetcherThread)
[2020-10-26 23:18:12,776] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=1] Partition t5_filtered-0 marked as failed (kafka.server.ReplicaFetcherThread)
[2020-10-26 23:18:12,848] INFO [ReplicaFetcher replicaId=0, leaderId=3, fetcherId=0] Partition t4-0 has an older epoch (0) than the current leader. Will await the new LeaderAndIsr state before resuming fetching. (kafka.server.ReplicaFetcherThread)
[2020-10-26 23:18:12,848] WARN [ReplicaFetcher replicaId=0, leaderId=3, fetcherId=0] Partition t4-0 marked as failed (kafka.server.ReplicaFetcherThread)
[2020-10-26 23:18:12,859] INFO [ReplicaFetcher replicaId=0, leaderId=4, fetcherId=1] Partition t1-0 has an older epoch (0) than the current leader. Will await the new LeaderAndIsr state before resuming fetching. (kafka.server.ReplicaFetcherThread)
[2020-10-26 23:18:12,860] WARN [ReplicaFetcher replicaId=0, leaderId=4, fetcherId=1] Partition t1-0 marked as failed (kafka.server.ReplicaFetcherThread)
[2020-10-26 23:18:12,965] INFO Session establishment complete on server zookeeper/10.233.2.139:2181, sessionid = 0x200002186240000, negotiated timeout = 3000 (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:18:12,999] INFO Stat of the created znode at /brokers/ids/0 is: 4294967559,4294967559,1603754292976,1603754292976,1,0,0,144115332060282880,294,0,4294967559
 (kafka.zk.KafkaZkClient)
[2020-10-26 23:18:12,999] INFO Registered broker 0 at path /brokers/ids/0 with addresses: ArrayBuffer(EndPoint(kafka-0.kafka.kafka.svc.cluster.local,9092,ListenerName(PLAINTEXT),PLAINTEXT), EndPoint(10.0.2.54,32400,ListenerName(OUTSIDE),PLAINTEXT)), czxid (broker epoch): 4294967559 (kafka.zk.KafkaZkClient)
[2020-10-26 23:18:13,070] INFO [ReplicaFetcherManager on broker 0] Removed fetcher for partitions Set(t5_filtered-0, t1_filtered-0, t4-0, t1-0, t4_warnings-0) (kafka.server.ReplicaFetcherManager)
[2020-10-26 23:18:13,078] INFO [ReplicaFetcherManager on broker 0] Added fetcher to broker 6 for partitions Map(t1_filtered-0 -> (offset=4184, leaderEpoch=1)) (kafka.server.ReplicaFetcherManager)
[2020-10-26 23:18:13,079] INFO [ReplicaFetcherManager on broker 0] Added fetcher to broker 4 for partitions Map(t1-0 -> (offset=4256, leaderEpoch=1)) (kafka.server.ReplicaFetcherManager)
[2020-10-26 23:18:13,080] INFO [ReplicaFetcherManager on broker 0] Added fetcher to broker 3 for partitions Map(t4-0 -> (offset=4311, leaderEpoch=1)) (kafka.server.ReplicaFetcherManager)
[2020-10-26 23:18:13,081] INFO [ReplicaFetcherManager on broker 0] Added fetcher to broker 1 for partitions Map(t4_warnings-0 -> (offset=0, leaderEpoch=1)) (kafka.server.ReplicaFetcherManager)
[2020-10-26 23:18:13,081] INFO [ReplicaFetcherManager on broker 0] Added fetcher to broker 1 for partitions Map(t5_filtered-0 -> (offset=4166, leaderEpoch=1)) (kafka.server.ReplicaFetcherManager)
[2020-10-26 23:18:13,083] INFO [ReplicaFetcher replicaId=0, leaderId=6, fetcherId=1] Starting (kafka.server.ReplicaFetcherThread)
[2020-10-26 23:18:13,088] INFO [Log partition=t5_filtered-0, dir=/var/lib/kafka/data/topics] Truncating to 4166 has no effect as the largest offset in the log is 4165 (kafka.log.Log)
[2020-10-26 23:18:13,088] INFO [Log partition=t1-0, dir=/var/lib/kafka/data/topics] Truncating to 4256 has no effect as the largest offset in the log is 4255 (kafka.log.Log)
[2020-10-26 23:18:13,089] INFO [Log partition=t4-0, dir=/var/lib/kafka/data/topics] Truncating to 4322 has no effect as the largest offset in the log is 4321 (kafka.log.Log)
[2020-10-26 23:18:13,107] INFO [Log partition=t1_filtered-0, dir=/var/lib/kafka/data/topics] Truncating to offset 4184 (kafka.log.Log)
[2020-10-26 23:18:13,108] INFO [Log partition=t1_filtered-0, dir=/var/lib/kafka/data/topics] Scheduling segments for deletion List() (kafka.log.Log)
[2020-10-26 23:18:13,113] INFO [Log partition=t1_filtered-0, dir=/var/lib/kafka/data/topics] Loading producer state till offset 4184 with message format version 2 (kafka.log.Log)
[2020-10-26 23:18:13,114] INFO [ProducerStateManager partition=t1_filtered-0] Writing producer snapshot at offset 4184 (kafka.log.ProducerStateManager)
[2020-10-26 23:18:13,179] INFO [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Truncating partition t4_warnings-0 to local high watermark 0 (kafka.server.ReplicaFetcherThread)
[2020-10-26 23:18:13,179] INFO [Log partition=t4_warnings-0, dir=/var/lib/kafka/data/topics] Truncating to 0 has no effect as the largest offset in the log is -1 (kafka.log.Log)
[2020-10-26 23:19:13,109] INFO [Log partition=t1_filtered-0, dir=/var/lib/kafka/data/topics] Deleting segments List() (kafka.log.Log)
[2020-10-26 23:19:14,418] INFO Unable to read additional data from server sessionid 0x200002186240000, likely server has closed socket, closing socket connection and attempting reconnect (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:19:15,882] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:19:15,890] INFO Socket connection established, initiating session, client: /10.233.122.2:53236, server: zookeeper/10.233.2.139:2181 (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:19:15,911] INFO Session establishment complete on server zookeeper/10.233.2.139:2181, sessionid = 0x200002186240000, negotiated timeout = 3000 (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:19:36,188] INFO Unable to read additional data from server sessionid 0x200002186240000, likely server has closed socket, closing socket connection and attempting reconnect (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:19:37,926] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:19:37,935] INFO Socket connection established, initiating session, client: /10.233.122.2:53322, server: zookeeper/10.233.2.139:2181 (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:19:37,944] INFO Session establishment complete on server zookeeper/10.233.2.139:2181, sessionid = 0x200002186240000, negotiated timeout = 3000 (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:23:03,834] INFO [ReplicaFetcherManager on broker 0] Removed fetcher for partitions Set(t1_filtered-0) (kafka.server.ReplicaFetcherManager)
[2020-10-26 23:23:03,835] INFO [Partition t1_filtered-0 broker=0] t1_filtered-0 starts at leader epoch 2 from offset 62451 with high watermark 62448. Previous leader epoch was 1. (kafka.cluster.Partition)
[2020-10-26 23:23:03,839] INFO [ReplicaFetcher replicaId=0, leaderId=6, fetcherId=1] Shutting down (kafka.server.ReplicaFetcherThread)
[2020-10-26 23:23:03,843] INFO [ReplicaFetcher replicaId=0, leaderId=6, fetcherId=1] Error sending fetch request (sessionId=757160850, epoch=6653) to node 6: {}. (org.apache.kafka.clients.FetchSessionHandler)
java.io.IOException: Client was shutdown before response was read
	at org.apache.kafka.clients.NetworkClientUtils.sendAndReceive(NetworkClientUtils.java:109)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:108)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:206)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:300)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:135)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:134)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:117)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:96)
[2020-10-26 23:23:03,850] INFO [ReplicaFetcher replicaId=0, leaderId=6, fetcherId=1] Stopped (kafka.server.ReplicaFetcherThread)
[2020-10-26 23:23:03,850] INFO [ReplicaFetcher replicaId=0, leaderId=6, fetcherId=1] Shutdown completed (kafka.server.ReplicaFetcherThread)
[2020-10-26 23:23:11,633] INFO [GroupMetadataManager brokerId=0] Removed 0 expired offsets in 0 milliseconds. (kafka.coordinator.group.GroupMetadataManager)
[2020-10-26 23:28:15,480] INFO [ReplicaFetcherManager on broker 0] Removed fetcher for partitions Set(t1-0) (kafka.server.ReplicaFetcherManager)
[2020-10-26 23:28:15,480] INFO [Partition t1-0 broker=0] t1-0 starts at leader epoch 2 from offset 123598 with high watermark 123598. Previous leader epoch was 1. (kafka.cluster.Partition)
[2020-10-26 23:28:15,483] INFO [ReplicaFetcher replicaId=0, leaderId=4, fetcherId=1] Shutting down (kafka.server.ReplicaFetcherThread)
[2020-10-26 23:28:15,484] INFO [ReplicaFetcher replicaId=0, leaderId=4, fetcherId=1] Error sending fetch request (sessionId=607137479, epoch=37690) to node 4: {}. (org.apache.kafka.clients.FetchSessionHandler)
java.io.IOException: Connection to 4 was disconnected before the response was read
	at org.apache.kafka.clients.NetworkClientUtils.sendAndReceive(NetworkClientUtils.java:100)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:108)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:206)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:300)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:135)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:134)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:117)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:96)
[2020-10-26 23:28:15,485] INFO [ReplicaFetcher replicaId=0, leaderId=4, fetcherId=1] Stopped (kafka.server.ReplicaFetcherThread)
[2020-10-26 23:28:15,486] INFO [ReplicaFetcher replicaId=0, leaderId=4, fetcherId=1] Shutdown completed (kafka.server.ReplicaFetcherThread)
[2020-10-26 23:29:01,611] INFO Unable to read additional data from server sessionid 0x200002186240000, likely server has closed socket, closing socket connection and attempting reconnect (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:29:09,347] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:29:10,375] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:29:11,903] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:29:12,934] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:29:14,797] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:29:15,814] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:29:17,353] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:29:18,375] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:29:19,854] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:29:20,871] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:29:22,535] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:29:23,558] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:29:24,761] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:29:25,798] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:29:26,917] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:29:27,942] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:29:29,157] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:29:30,182] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:29:31,464] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:29:32,490] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:29:34,100] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:29:35,111] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:29:42,550] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:29:43,563] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:29:45,269] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:29:46,279] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:29:48,081] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:29:49,098] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:29:50,792] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:29:51,815] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:29:53,862] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:29:54,887] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:29:56,556] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:29:57,574] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:29:58,721] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:29:59,751] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:30:00,898] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:30:01,926] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:30:03,036] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:30:04,071] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:30:05,627] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:30:06,663] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:30:14,503] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:30:15,526] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:30:16,812] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:30:17,831] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:30:19,045] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:30:20,070] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:30:21,328] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:30:22,342] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:30:24,163] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:30:25,190] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:30:26,494] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:30:27,526] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:30:28,705] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:30:29,734] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:30:30,878] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:30:31,910] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:30:33,953] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:30:34,986] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:30:36,216] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:30:37,254] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:30:38,692] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:30:39,718] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:30:42,582] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:30:43,622] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:30:44,885] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:30:45,899] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:30:47,520] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:30:48,550] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:30:50,140] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:30:51,174] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:30:52,477] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:30:53,510] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:30:55,155] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:30:56,166] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:30:57,886] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:30:58,918] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:31:00,208] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:31:01,222] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:31:02,541] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:31:02,546] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:31:03,777] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:31:04,806] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:31:05,924] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:31:06,951] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:31:08,387] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:31:09,414] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:31:10,976] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:31:12,007] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:31:18,326] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:31:19,338] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:31:21,156] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:31:22,182] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:31:24,049] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:31:25,062] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:31:26,695] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:31:27,718] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:31:29,038] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:31:30,054] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:31:31,393] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:31:32,422] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:31:33,867] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:31:34,886] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:31:36,833] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:31:37,862] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:31:39,594] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:31:40,614] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:31:42,265] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:31:43,302] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:31:49,753] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:31:50,803] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:31:51,956] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:31:52,966] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:31:54,139] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:31:55,174] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:31:56,741] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:31:57,767] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:31:59,430] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:32:00,454] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:32:01,965] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:32:02,982] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:32:04,431] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:32:05,447] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:32:07,251] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:32:08,262] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:32:09,557] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:32:10,665] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:32:11,767] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:32:12,807] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:32:14,513] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:32:15,527] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:32:21,395] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:32:22,406] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:32:24,119] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:32:25,158] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:32:26,683] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:32:27,723] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:32:29,147] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:32:30,182] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:32:31,454] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:32:32,486] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:32:34,401] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:32:35,432] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:32:37,524] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:32:38,535] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:32:40,512] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:32:41,542] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:32:43,467] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:32:44,486] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:32:45,797] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:32:46,826] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:32:52,475] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:32:53,511] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:32:55,069] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:32:56,103] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:32:57,908] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:32:58,922] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:33:00,965] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:33:01,991] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:33:03,631] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:33:04,647] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:33:06,618] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:33:07,659] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:33:08,780] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:33:09,798] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:33:11,633] INFO [GroupMetadataManager brokerId=0] Removed 0 expired offsets in 0 milliseconds. (kafka.coordinator.group.GroupMetadataManager)
[2020-10-26 23:33:11,772] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:33:12,806] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:33:14,774] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:33:15,783] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:33:17,769] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:33:18,794] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:33:24,574] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:33:25,611] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:33:27,372] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:33:28,391] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:33:29,661] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:33:30,695] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:33:32,372] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:33:33,383] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:33:34,920] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:33:35,942] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:33:37,146] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:33:38,182] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:33:39,460] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:33:40,486] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:33:41,775] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:33:42,790] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:33:44,673] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:33:45,707] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:33:47,475] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:33:48,487] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:33:50,503] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:33:51,526] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:33:53,263] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:33:54,282] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:33:56,145] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:33:57,158] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:33:58,440] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:33:59,462] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:34:00,732] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:34:01,766] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:34:03,674] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:34:04,710] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:34:06,125] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:34:07,142] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:34:08,314] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:34:09,355] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:34:10,957] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:34:11,975] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:34:13,791] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:34:14,826] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:34:16,787] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:34:17,802] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:34:18,993] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:34:20,006] INFO Socket error occurred: zookeeper/10.233.2.139:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:34:21,535] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:34:21,544] INFO Socket connection established, initiating session, client: /10.233.122.2:58654, server: zookeeper/10.233.2.139:2181 (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:34:21,564] WARN Unable to reconnect to ZooKeeper service, session 0x200002186240000 has expired (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:34:21,564] INFO Unable to reconnect to ZooKeeper service, session 0x200002186240000 has expired, closing socket connection (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:34:21,564] INFO [ZooKeeperClient Kafka server] Session expired. (kafka.zookeeper.ZooKeeperClient)
[2020-10-26 23:34:21,564] INFO EventThread shut down for session: 0x200002186240000 (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:34:21,567] INFO [ZooKeeperClient Kafka server] Initializing a new session to zookeeper:2181. (kafka.zookeeper.ZooKeeperClient)
[2020-10-26 23:34:21,567] INFO Initiating client connection, connectString=zookeeper:2181 sessionTimeout=5000 watcher=kafka.zookeeper.ZooKeeperClient$ZooKeeperClientWatcher$@2fd953a6 (org.apache.zookeeper.ZooKeeper)
[2020-10-26 23:34:21,568] INFO jute.maxbuffer value is 4194304 Bytes (org.apache.zookeeper.ClientCnxnSocket)
[2020-10-26 23:34:21,568] INFO zookeeper.request.timeout value is 0. feature enabled= (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:34:21,569] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:34:21,570] INFO Creating /brokers/ids/0 (is it secure? false) (kafka.zk.KafkaZkClient)
[2020-10-26 23:34:21,578] INFO Socket connection established, initiating session, client: /10.233.122.2:58656, server: zookeeper/10.233.2.139:2181 (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:34:21,615] INFO Session establishment complete on server zookeeper/10.233.2.139:2181, sessionid = 0x200003135e20004, negotiated timeout = 3000 (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:34:21,645] INFO Stat of the created znode at /brokers/ids/0 is: 98784247819,98784247819,1603755261626,1603755261626,1,0,0,144115399433256964,294,0,98784247819
 (kafka.zk.KafkaZkClient)
[2020-10-26 23:34:21,645] INFO Registered broker 0 at path /brokers/ids/0 with addresses: ArrayBuffer(EndPoint(kafka-0.kafka.kafka.svc.cluster.local,9092,ListenerName(PLAINTEXT),PLAINTEXT), EndPoint(10.0.2.54,32400,ListenerName(OUTSIDE),PLAINTEXT)), czxid (broker epoch): 98784247819 (kafka.zk.KafkaZkClient)
[2020-10-26 23:34:21,897] INFO [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Partition t4_warnings-0 has an older epoch (1) than the current leader. Will await the new LeaderAndIsr state before resuming fetching. (kafka.server.ReplicaFetcherThread)
[2020-10-26 23:34:21,898] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Partition t4_warnings-0 marked as failed (kafka.server.ReplicaFetcherThread)
[2020-10-26 23:34:22,238] INFO [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=1] Partition t5_filtered-0 has an older epoch (1) than the current leader. Will await the new LeaderAndIsr state before resuming fetching. (kafka.server.ReplicaFetcherThread)
[2020-10-26 23:34:22,238] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=1] Partition t5_filtered-0 marked as failed (kafka.server.ReplicaFetcherThread)
[2020-10-26 23:34:22,266] INFO [ReplicaFetcherManager on broker 0] Removed fetcher for partitions Set(t5_filtered-0, t1_filtered-0, t4-0, t4_warnings-0) (kafka.server.ReplicaFetcherManager)
[2020-10-26 23:34:22,268] INFO [ReplicaFetcherManager on broker 0] Added fetcher to broker 1 for partitions Map(t4_warnings-0 -> (offset=3939, leaderEpoch=3)) (kafka.server.ReplicaFetcherManager)
[2020-10-26 23:34:22,274] INFO [Log partition=t4_warnings-0, dir=/var/lib/kafka/data/topics] Truncating to 3939 has no effect as the largest offset in the log is 3938 (kafka.log.Log)
[2020-10-26 23:34:22,286] INFO [ReplicaFetcherManager on broker 0] Added fetcher to broker 2 for partitions Map(t4-0 -> (offset=199182, leaderEpoch=4)) (kafka.server.ReplicaFetcherManager)
[2020-10-26 23:34:22,287] INFO [ReplicaFetcherManager on broker 0] Added fetcher to broker 1 for partitions Map(t5_filtered-0 -> (offset=198238, leaderEpoch=3)) (kafka.server.ReplicaFetcherManager)
[2020-10-26 23:34:22,292] INFO [Log partition=t5_filtered-0, dir=/var/lib/kafka/data/topics] Truncating to 198238 has no effect as the largest offset in the log is 198237 (kafka.log.Log)
[2020-10-26 23:34:22,296] INFO [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Starting (kafka.server.ReplicaFetcherThread)
[2020-10-26 23:34:22,311] INFO [ReplicaFetcherManager on broker 0] Added fetcher to broker 2 for partitions Map(t1_filtered-0 -> (offset=198560, leaderEpoch=5)) (kafka.server.ReplicaFetcherManager)
[2020-10-26 23:34:22,312] INFO [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=1] Starting (kafka.server.ReplicaFetcherThread)
[2020-10-26 23:34:22,313] INFO [ReplicaFetcher replicaId=0, leaderId=3, fetcherId=0] Shutting down (kafka.server.ReplicaFetcherThread)
[2020-10-26 23:34:22,313] INFO [ReplicaFetcher replicaId=0, leaderId=3, fetcherId=0] Stopped (kafka.server.ReplicaFetcherThread)
[2020-10-26 23:34:22,313] INFO [ReplicaFetcher replicaId=0, leaderId=3, fetcherId=0] Shutdown completed (kafka.server.ReplicaFetcherThread)
[2020-10-26 23:34:22,321] INFO [Log partition=t4-0, dir=/var/lib/kafka/data/topics] Truncating to offset 199182 (kafka.log.Log)
[2020-10-26 23:34:22,322] INFO [Log partition=t4-0, dir=/var/lib/kafka/data/topics] Scheduling segments for deletion List() (kafka.log.Log)
[2020-10-26 23:34:22,324] INFO [Log partition=t4-0, dir=/var/lib/kafka/data/topics] Loading producer state till offset 199182 with message format version 2 (kafka.log.Log)
[2020-10-26 23:34:22,325] INFO [ProducerStateManager partition=t4-0] Writing producer snapshot at offset 199182 (kafka.log.ProducerStateManager)
[2020-10-26 23:34:22,333] INFO [Log partition=t1_filtered-0, dir=/var/lib/kafka/data/topics] Truncating to offset 198560 (kafka.log.Log)
[2020-10-26 23:34:22,333] INFO [Log partition=t1_filtered-0, dir=/var/lib/kafka/data/topics] Scheduling segments for deletion List() (kafka.log.Log)
[2020-10-26 23:34:22,335] INFO [Log partition=t1_filtered-0, dir=/var/lib/kafka/data/topics] Loading producer state till offset 198560 with message format version 2 (kafka.log.Log)
[2020-10-26 23:34:22,337] INFO [ProducerStateManager partition=t1_filtered-0] Loading producer state from snapshot file '/var/lib/kafka/data/topics/t1_filtered-0/00000000000000004184.snapshot' (kafka.log.ProducerStateManager)
[2020-10-26 23:34:22,346] INFO [ProducerStateManager partition=t1_filtered-0] Writing producer snapshot at offset 198560 (kafka.log.ProducerStateManager)
[2020-10-26 23:34:22,760] INFO [ReplicaFetcherManager on broker 0] Removed fetcher for partitions Set(t1-0) (kafka.server.ReplicaFetcherManager)
[2020-10-26 23:34:22,769] INFO [ReplicaFetcherManager on broker 0] Added fetcher to broker 5 for partitions Map(t1-0 -> (offset=198437, leaderEpoch=5)) (kafka.server.ReplicaFetcherManager)
[2020-10-26 23:34:22,775] INFO [ReplicaFetcher replicaId=0, leaderId=5, fetcherId=1] Starting (kafka.server.ReplicaFetcherThread)
[2020-10-26 23:34:22,799] INFO [ReplicaFetcher replicaId=0, leaderId=5, fetcherId=1] Retrying leaderEpoch request for partition t1-0 as the leader reported an error: UNKNOWN_LEADER_EPOCH (kafka.server.ReplicaFetcherThread)
[2020-10-26 23:34:23,813] INFO [Log partition=t1-0, dir=/var/lib/kafka/data/topics] Truncating to 198437 has no effect as the largest offset in the log is 198436 (kafka.log.Log)
[2020-10-26 23:34:27,143] INFO [ReplicaFetcherManager on broker 0] Removed fetcher for partitions Set(t1_filtered-0) (kafka.server.ReplicaFetcherManager)
[2020-10-26 23:34:27,144] INFO [Partition t1_filtered-0 broker=0] t1_filtered-0 starts at leader epoch 6 from offset 199534 with high watermark 199534. Previous leader epoch was 5. (kafka.cluster.Partition)
[2020-10-26 23:34:27,145] INFO [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=1] Shutting down (kafka.server.ReplicaFetcherThread)
[2020-10-26 23:34:27,149] INFO [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=1] Error sending fetch request (sessionId=591387061, epoch=75) to node 2: {}. (org.apache.kafka.clients.FetchSessionHandler)
java.io.IOException: Client was shutdown before response was read
	at org.apache.kafka.clients.NetworkClientUtils.sendAndReceive(NetworkClientUtils.java:109)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:108)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:206)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:300)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:135)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:134)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:117)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:96)
[2020-10-26 23:34:27,149] INFO [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=1] Shutdown completed (kafka.server.ReplicaFetcherThread)
[2020-10-26 23:34:27,149] INFO [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=1] Stopped (kafka.server.ReplicaFetcherThread)
[2020-10-26 23:34:27,355] INFO [ReplicaFetcherManager on broker 0] Removed fetcher for partitions Set(t4-0) (kafka.server.ReplicaFetcherManager)
[2020-10-26 23:34:27,358] INFO [ReplicaFetcherManager on broker 0] Added fetcher to broker 3 for partitions Map(t4-0 -> (offset=200196, leaderEpoch=5)) (kafka.server.ReplicaFetcherManager)
[2020-10-26 23:34:27,359] INFO [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Shutting down (kafka.server.ReplicaFetcherThread)
[2020-10-26 23:34:27,359] INFO [ReplicaFetcher replicaId=0, leaderId=3, fetcherId=0] Starting (kafka.server.ReplicaFetcherThread)
[2020-10-26 23:34:27,360] INFO [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error sending fetch request (sessionId=1998418102, epoch=117) to node 2: {}. (org.apache.kafka.clients.FetchSessionHandler)
java.io.IOException: Client was shutdown before response was read
	at org.apache.kafka.clients.NetworkClientUtils.sendAndReceive(NetworkClientUtils.java:109)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:108)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:206)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:300)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:135)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:134)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:117)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:96)
[2020-10-26 23:34:27,360] INFO [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Stopped (kafka.server.ReplicaFetcherThread)
[2020-10-26 23:34:27,361] INFO [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Shutdown completed (kafka.server.ReplicaFetcherThread)
[2020-10-26 23:34:27,383] INFO [Log partition=t4-0, dir=/var/lib/kafka/data/topics] Truncating to 200196 has no effect as the largest offset in the log is 200195 (kafka.log.Log)
[2020-10-26 23:34:27,413] INFO [ReplicaFetcherManager on broker 0] Removed fetcher for partitions Set(t1-0) (kafka.server.ReplicaFetcherManager)
[2020-10-26 23:34:27,416] INFO [ReplicaFetcherManager on broker 0] Added fetcher to broker 4 for partitions Map(t1-0 -> (offset=199500, leaderEpoch=6)) (kafka.server.ReplicaFetcherManager)
[2020-10-26 23:34:27,416] INFO [ReplicaFetcher replicaId=0, leaderId=5, fetcherId=1] Shutting down (kafka.server.ReplicaFetcherThread)
[2020-10-26 23:34:27,416] INFO [ReplicaFetcher replicaId=0, leaderId=5, fetcherId=1] Error sending fetch request (sessionId=1904868180, epoch=120) to node 5: {}. (org.apache.kafka.clients.FetchSessionHandler)
java.io.IOException: Client was shutdown before response was read
	at org.apache.kafka.clients.NetworkClientUtils.sendAndReceive(NetworkClientUtils.java:109)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:108)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:206)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:300)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:135)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:134)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:117)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:96)
[2020-10-26 23:34:27,416] INFO [ReplicaFetcher replicaId=0, leaderId=5, fetcherId=1] Stopped (kafka.server.ReplicaFetcherThread)
[2020-10-26 23:34:27,416] INFO [ReplicaFetcher replicaId=0, leaderId=5, fetcherId=1] Shutdown completed (kafka.server.ReplicaFetcherThread)
[2020-10-26 23:34:27,422] INFO [ReplicaFetcher replicaId=0, leaderId=4, fetcherId=1] Starting (kafka.server.ReplicaFetcherThread)
[2020-10-26 23:34:27,439] INFO [ReplicaFetcher replicaId=0, leaderId=4, fetcherId=1] Retrying leaderEpoch request for partition t1-0 as the leader reported an error: UNKNOWN_LEADER_EPOCH (kafka.server.ReplicaFetcherThread)
[2020-10-26 23:34:28,444] INFO [Log partition=t1-0, dir=/var/lib/kafka/data/topics] Truncating to 199500 has no effect as the largest offset in the log is 199499 (kafka.log.Log)
[2020-10-26 23:35:17,812] INFO Unable to read additional data from server sessionid 0x200003135e20004, likely server has closed socket, closing socket connection and attempting reconnect (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:35:18,971] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:35:18,980] INFO Socket connection established, initiating session, client: /10.233.122.2:58910, server: zookeeper/10.233.2.139:2181 (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:35:19,004] INFO Session establishment complete on server zookeeper/10.233.2.139:2181, sessionid = 0x200003135e20004, negotiated timeout = 3000 (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:35:22,323] INFO [Log partition=t4-0, dir=/var/lib/kafka/data/topics] Deleting segments List() (kafka.log.Log)
[2020-10-26 23:35:22,333] INFO [Log partition=t1_filtered-0, dir=/var/lib/kafka/data/topics] Deleting segments List() (kafka.log.Log)
[2020-10-26 23:35:52,598] INFO Unable to read additional data from server sessionid 0x200003135e20004, likely server has closed socket, closing socket connection and attempting reconnect (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:35:54,394] INFO Opening socket connection to server zookeeper/10.233.2.139:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:35:54,403] INFO Socket connection established, initiating session, client: /10.233.122.2:59046, server: zookeeper/10.233.2.139:2181 (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:35:54,669] INFO Session establishment complete on server zookeeper/10.233.2.139:2181, sessionid = 0x200003135e20004, negotiated timeout = 3000 (org.apache.zookeeper.ClientCnxn)
[2020-10-26 23:35:56,248] INFO [ReplicaFetcher replicaId=0, leaderId=4, fetcherId=1] Partition t1-0 has an older epoch (6) than the current leader. Will await the new LeaderAndIsr state before resuming fetching. (kafka.server.ReplicaFetcherThread)
[2020-10-26 23:35:56,249] WARN [ReplicaFetcher replicaId=0, leaderId=4, fetcherId=1] Partition t1-0 marked as failed (kafka.server.ReplicaFetcherThread)
[2020-10-26 23:35:56,461] INFO [ReplicaFetcherManager on broker 0] Removed fetcher for partitions Set(t1-0) (kafka.server.ReplicaFetcherManager)
[2020-10-26 23:35:56,462] INFO [ReplicaFetcherManager on broker 0] Added fetcher to broker 4 for partitions Map(t1-0 -> (offset=217275, leaderEpoch=7)) (kafka.server.ReplicaFetcherManager)
[2020-10-26 23:35:56,467] INFO [Log partition=t1-0, dir=/var/lib/kafka/data/topics] Truncating to 217275 has no effect as the largest offset in the log is 217274 (kafka.log.Log)
